src/sql/src/plan/statement/ddl.rs

```diff
@@ -1143,10 +1143,28 @@ fn compile_proto(schema: &str) -> Result<Vec<u8>, anyhow::Error> {
         .with_context(|| format!("creating out directory {}", out_dir.display()))?;
 
     // Compile schema string.
-    protoc::Protoc::new()
+    match protoc::Protoc::new()
         .include(&include_dir)
         .input(file.path())
-        .compile_into(&out_dir)?;
+        .compile_into(&out_dir)
+    {
+        Ok(()) => (),
+        Err(e) => {
+            // Make protobuf import errors more user-friendly.
+            if let Some(cap) =
+                Regex::new(r#"protobuf path \\"(?P<reference>.*)\\" is not found in import path"#)
```

Comment:
This is a good opportunity to extract a lazy static to avoid recompiling the regex on every pass through the function:

```rust
lazy_static! {
    static ref MISSING_IMPORT_ERROR: Regex = Regex::new("...").expect("known valid");
}

if let Some(captures) = MISSING_IMPORT_ERROR {

}
```